klog.go

```diff
@@ -1098,13 +1147,23 @@ func (l *loggingT) setV(pc uintptr) Level {
 
 // Verbose is a boolean type that implements Infof (like Printf) etc.
 // See the documentation of V for more information.
-type Verbose bool
+type Verbose struct {
```

Comment:
note that this broke consumers like client-go. xref https://github.com/kubernetes/client-go/issues/654 https://github.com/kubernetes/client-go/issues/656 https://github.com/kubernetes/client-go/issues/652

Reply:
IMO klog needs to follow real semver rules. It's not in our staging/ directory, there is no reason not to.